app/models/bike.rb

```diff
@@ -433,7 +433,9 @@ def visible_by(passed_user = nil)
   end
 
   def find_current_stolen_record
-    stolen_records.last if stolen_records.any?
+    return unless stolen_records.any?
+    self.current_stolen_record = stolen_records.last
+    current_stolen_record
```

Comment:
Nit: is `current_stolen_record` necessary? Won't the previous line return that anyway?